modules/test/test-howtos.bats

```diff
@@ -52,15 +52,6 @@ load 'test/test_helper.bash'
 }
 
 @test "[howtos] - KvOperations.java" {
```

Comment:
We no longer need to skip this as it will be tested against the latest Couchbase Server build and pass - once we merge into the default branch.